alma/controllers/hook/GetContentHookController.php

```diff
@@ -166,10 +178,15 @@ public function processConfiguration()
                     'string' => Tools::getValue(ExcludedCategoryAdminFormBuilder::ALMA_NOT_ELIGIBLE_CATEGORIES . '_' . $language['id_lang']),
                 ];
 
-                if (empty($titles[$language['id_lang']]['string'])
-                    || empty($descriptions[$language['id_lang']]['string'])
+                if (
```

Comment:
à découper du genre
				if($this->isEmptyLanguage($language['id_lang'])) {
					.....
				}
		protected function isEmptyLanguage($languageId) {
			return 				empty($titles[$languageId])
				|| empty($titlesPayNow[$languageId]['string'])
				|| empty($titlesDeferred[$languageId]['string'])
				|| empty($titlesCredit[$languageId]['string'])
				|| empty($descriptionsPayNow[$languageId]['string'])
				|| empty($descriptions[$languageId]['string'])
				|| empty($descriptionsDeferred[$languageId]['string'])
				|| empty($descriptionsCredit[$languageId]['string']);
		}

Reply:
La méthode ne peux pas fonctionner comme ça, à moins d'envoyer toutes les variables à la fonction non ?